lamps-rfc7030-csrattrs.mkd

```diff
@@ -338,7 +340,7 @@ this means that the client SHOULD fill in the respective `GeneralName` value.
      extnID      EXTENSION.&id({ExtensionSet}),
      critical    BOOLEAN DEFAULT FALSE,
      extnValue   OCTET STRING (CONTAINING
-                 EXTENSION.&ExtnType({ExtensionSet}{@extnID})) OPTIONAL
+                 EXTENSION.&ExtnType({ExtensionSe}{@extnID})) OPTIONAL
```

Comment:
seems wrong to cut the "t"?

Reply:
Oops, this must have been due to an inadvertent keystroke in my editor.